swim/node.go

```diff
@@ -154,6 +162,7 @@ type Node struct {
 
 	logger log.Logger
 	log    log.Logger // wraps the provided logger with a 'local: address' field
+	clock  Clock
```

Comment:
I'll add some comments here saying that this clock is used to generate the incarnation numbers